src/acl.c

```diff
@@ -2566,17 +2566,17 @@ int ACLSaveToFile(const char *filename) {
 
 /* This function is called once the server is already running, modules are
  * loaded, and we are ready to start, in order to load the ACLs either from
- * the pending list of users defined in redis.conf, or from the ACL file.
+ * the pending list of users defined in placeholderkv.conf, or from the ACL file.
  * The function will just exit with an error if the user is trying to mix
  * both the loading methods. */
 void ACLLoadUsersAtStartup(void) {
     if (server.acl_filename[0] != '\0' && listLength(UsersToLoad) != 0) {
         serverLog(LL_WARNING,
-            "Configuring Redis with users defined in redis.conf and at "
```

Comment:
log line changes can be breaking - not particularly this line but in general. 

also the changes can be too extensive so that brings the concern of losing real change history. can we punt all log line changes into another discussion? we could probably live with "redis" in log lines in the first 1 or 2 releases.

A smaller change would also help with the cherry-picking effort when we release the "compatible" 7.2.4.

Reply:
Makes sense.